core/src/main/java/tech/pegasys/eth2signer/core/Runner.java

```diff
@@ -165,6 +184,16 @@ private String formatBlsSignature(final BlsArtifactSignature signature) {
     return signature.getSignatureData().toString();
   }
 
+  private String formatSecpSignature(final SecpArtifactSignature signature) {
```

Comment:
static?

Reply:
Not seeing the benefit making static since this a private method. Can of course, is there a good reason to make it static?